### PR TITLE
for issue #287; systemd + adding Juju topics

### PR DESCRIPTION
**Pull request by petermatulis**

### CODE REVIEW

File: src/en/juju-systemd.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# Juju and systemd
+
+Below we cover the important issue of the migration of the init system used in
+Ubuntu: upstart to systemd.
+
+See [https://wiki.ubuntu.com/SystemdForUpstartUsers](https://wiki.ubuntu.com/SystemdForUpstartUsers).
+
+
+## General
+
```
Comment (cherylj) [inline src/en/juju-systemd.md]: There is currently a bug where users cannot use vivid and upstart with Juju for the local provider:  https://bugs.launchpad.net/juju-core/+bug/1450092

Comment (petermatulis) [inline src/en/juju-systemd.md]: Ouch :skull:

**Comment by petermatulis** [comment]
@evilnick 
Edited as per IRC (avoiding Vivid)

**Comment by petermatulis** [comment]
I changed this to use the 'Manage Juju' header. Before I did that I did:

git pull https://github.com/juju/docs master

but I do not see my changes under 'Files changed'.

**Comment by petermatulis** [comment]
OK, it should be good now.

**Pull request merged**